app/helpers/menus_helper.rb

```diff
@@ -0,0 +1,13 @@
+module MenusHelper
+  # TODO: read about pundit
+
+  def food_items_for_select(food_items)
+    food_items.map { |item| [item.name, item.id, data: { price: item.price }] }
+  end
+
+  def user_role_status current_user
+    return 'You are admin of menu' if !current_user.nil? && current_user.has_role?(:admin)
```

Comment:
`current_user&.has_role?(:admin)`

[source](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/)

+ locales